config/default.json

```diff
@@ -6,7 +6,7 @@
         "codegenExecRelPath": "swagger-codegen-cli.jar",
         "codegenRepoUrl": "git@github.com:swagger-api/swagger-codegen.git",
         "codegenTemplatePath": "swagger-codegen/lib/javascript-template",
-        "clientDestPath": "dist/client/codegen-client"
+        "codeGenCompileDest": "dist/client/codegen-client-compiled"
```

Comment:
this was very confusing.  compilation is a two part process. first, codegen => commonjs ready, second => webpack'd.  so, dest-path is now disambiguated to specify that this path is specifically for codegen-compiled output